cmd/exchange_wrapper_standards/exchange_wrapper_standards_test.go

```diff
@@ -598,6 +604,11 @@ var blockedCIExchanges = []string{
 	"bybit",   // bybit API is banned from executing within the US where github Actions is ran
 }
 
+// blockedExchanges are exchanges that are not able to be tested in general
+var blockedExchanges = []string{
+	"coinbasepro", // coinbasepro API requires authentication for almost every endpoint
+}
+
```

Comment:
Maybe you've thought about this already, but worth mentioning in case it hasn't been covered already:
As a design pattern I think it's preferable to find any way to consolidate all information about an exchange onto it's own packages.

Off the top of my head I'm thinking about adding an exported constant using build tags for tests, and then using reflect to check for it.
The things I'm aiming for with that are:
* No cost or boilerplate to exchanges which don't care
* No production runtime cost
* More information about the exchange is consolidated into it's dir, making it easier to find and delete in one go

I'm not suggesting you need to do this, especially in light of `blockedCIExchanges` already existing, but wanted to share it in case you want to engage, and for any other work in flight where it's appropriate.

Reply:
After incorporating public endpoints from an earlier version of the API, I no longer need that blockedExchanges variable to exist so I've removed it.

Moving towards that sort of design for other similar stuff within those standards seems like a good idea, but I think it's a bit out of scope for my current work.